proc_CPMG.py

```diff
@@ -90,20 +58,17 @@ def print_fun(x, f, accepted):
     phshift = exp(-1j*2*pi*f_axis*(firstorder*1e-6))
     phshift *= exp(-1j*2*pi*zeroorder_rad)
     s *= phshift
-    print("RELATIVE PHASE SHIFT WAS {:0.1f}\\us and {:0.1f}$^\circ$".format(
-            firstorder,angle(zeroorder_rad)/pi*180))
+    logger.info(strm("RELATIVE PHASE SHIFT WAS %0.1f\\us and %0.1f$^\circ$".format(firstorder,angle(zeroorder_rad)/pi*180)))
```

Comment:
To make it easier to find, I reference this as a single comment [original conversation here](https://github.com/jmfrancklab/proc_scripts/pull/5#discussion_r430549214)

I'm saying I would try the following to debug:
OK -- I would try this to debug:

- assign the result of the format statement to a string variable
- use print to see that the string variable is equal to what it's supposed to be (that the format statement is working properly)
- call logger.info with the variable as an argument, and see whether or not it prints
- call logger.info in the same place with a standard string (e.g. "hello") as an argument to see whether or not it prints

Reply:
On slack, it sounds like this is an issue with setting up the logger

[This](https://stackoverflow.com/questions/14058453/making-python-loggers-output-all-messages-to-stdout-in-addition-to-log-file) is a good discussion on logging.

If you want `logger.info` to print to the command line, and it's not, then you need to set up the logger differently.

Note that you are *totally* encouraged to make a pull request that modifies `init_logging` in `general_functions` of pyspecdata to make life easier across the board.

Reply:
this is fixed